test/libsolidity/smtCheckerTests/imports/import_base.sol

```diff
@@ -0,0 +1,25 @@
+==== Source: base ====
+contract Base {
+	uint x;
+	address a;
+	function f() internal returns (uint) {
+		a = address(this);
+		++x;
+		return 2;
+	}
+}
+==== Source: der ====
+pragma experimental SMTChecker;
+import "base";
+contract Der is Base {
+	function g(uint y) public {
+		x += f();
+		assert(y > x);
+	}
+}
+// ----
+// Warning 1218: (der:101-109): Error trying to invoke SMT solver.
```

Comment:
Is this expected?

Reply:
Can happen when CHC checks for underflow/overflow here. I still really dislike this message, but z3 throws when `rlimit` is over...